### PR TITLE
Objects Automated Test - LPS-145718 | master

### PR DESCRIPTION
**Pull request by andreefarias**
https://issues.liferay.com/browse/LPS-145718

### CODE REVIEW

**Comment by andreefarias** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase [hunk 1]
```diff
@@ -51,13 +51,32 @@ definition {
...
+			actionName = "Action Name",
+			thenAction = "Webhook",
+			urlAction = "http://localhost:8080",
+			whenAction = "On After Add");
+
+		ObjectAdmin.assertObjectActions(
+			actionActive = "Yes",
+			actionName = "Action Name");
 
+		ObjectAdmin.deleteActionViaUI(actionName = "Action Name");
```
Comment (rodrigocunhaa) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase:79]: After you delete, you have to also assert that the Action is not present.

**Review by rodrigocunhaa — CHANGES_REQUESTED** [review]
Hi @andreefarias ,

Please, check my comments and make the necessary changes. Thanks!

**Comment by andreefarias** [comment]
ci:test:sf

**Comment by rodrigocunhaa** [comment]
ci:forward

**Pull request closed without merging**